### DISCUSSION TITLE
How to get the version of a plausible docker image

### DISCUSSION THREAD

**Discussion by devrsi0n** [plausible/analytics#1614] [Self-Hosted Support] [answered]
Hi folks, I want to know how to get the version of a plausible docker image? Currently, I'm running a plausible instance and plausible frontend code for [Chirpy](https://github.com/devrsi0n/chirpy), but there are some issues caused by server & client version mismatching. I want to make sure I'm using the correct version of instance. Could you please guide me on this, thanks 🙏

The output of `docker images`:
![image](https://user-images.githubusercontent.com/7880675/149780822-90065391-3472-4b9a-ac60-db6423cf7a3a.png)
I can't find the accurate version, e.g., `v.1.4.3`.

**Comment by ukutaht** [comment, marked as answer]
I would recommend pinning the version down in your docker-compose file. So instead of using `latest` you could use the tag `v1.4.3` and know exactly what you're getting.

**Reply by devrsi0n** [reply]
@ukutaht Thanks, that's exactly what I'm doing. It would be a valuable feature if we can get the version on web UI.

**Reply by ukutaht** [reply]
Thanks! I'll add it to the list of improvements for self-hosted releases

**Reply by devrsi0n** [reply]
@ukutaht Thank you. ❤️
